linkml/generators/jsonldcontextgen.py

```diff
@@ -130,19 +130,19 @@ def end_schema(
             print(as_json(context))
 
     def visit_class(self, cls: ClassDefinition) -> bool:
+        if cls.mixin:
```

Comment:
this seems like a separate concern from the main intent of the PR?

Reply:
I had the feeling it was linked to fixing class and slot generation by the JSON-LD context generator, this can be another PR if needed 😊